src/com/twitter/intellij/pants/service/PantsCompileOptionsExecutor.java

```diff
@@ -169,12 +169,12 @@ private String loadProjectStructureFromTargets(
   ) throws IOException, ExecutionException {
     final File outputFile = FileUtil.createTempFile("pants_depmap_run", ".out");
     final GeneralCommandLine command = getCommand(outputFile, statusConsumer);
-    statusConsumer.consume("Resolving dependencies...");
+    statusConsumer.consume("Exporting Pants project into IDEA format...");
     PantsMetrics.markExportStart();
-    final ProcessOutput processOutput = getProcessOutput(command, processAdapter);
+    final ProcessOutput processOutput = getProcessOutput(command);
```

Comment:
any reason you are removing the `processAdapter`?

Reply:
Yes -- it's because the `processAdapter` argument wasn't being used in the `getProcessOutput()` method, so I removed that argument and the argument from all of its callers (just here, I think).